src/game/models/asteroid.rs

```diff
@@ -59,19 +60,31 @@ fn generate_jagged_shape() -> CircularPolygon {
 
 impl Asteroid {
     pub fn new(window_size: Size) -> Asteroid {
+        let radius = cmp::max(window_size.width, window_size.height) as f64 + RADIUS;
+        let angle = f64::consts::PI * 2.0 * rand::random::<f64>();
```

Comment:
Should we define `TAU` or `2_PI` somewhere? We use it in so many places. Maybe `models/mod.rs`? We use it in several models.